src/third_party/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ endif ()
 
 if(ENABLE_TRACING)
   target_sources(third_party_lib PRIVATE minitrace.c)
+  set_source_files_properties(minitrace.c PROPERTIES COMPILE_DEFINITIONS "MTR_ENABLED;_POSIX_C_SOURCE=200809L")
```

Comment:
I think that it would be better to apply `config.h` instead of adding this special case, but I'll make that as a follow-up commit.